src/common/containers/build-details.js

```diff
@@ -92,23 +91,14 @@ class BuildDetails extends Component {
         }
         { build &&
           <div>
-            <Table>
-              <Head>
-                <Row>
-                  <Header col="20">Number</Header>
-                  <Header col="20">Architecture</Header>
-                  <Header col="20">Duration</Header>
-                  <Header col="40">Result</Header>
-                </Row>
-              </Head>
-              <Body>
-                <BuildRow
-                  repository={repository}
-                  {...build}
-                  isLinked={false} // disable link to the same build we are already looking at
-                />
-              </Body>
-            </Table>
+            <BuildsList
+              repository={repository}
+              builds={[{
+                ...build,
+                // Disable link to the same build we're already looking at.
+                isLinked: false
```

Comment:
This part is a little bit hacky. It depends on internal implementation of `BuildsList` and the fact that it passes whole build object to `BuildRow` as props.

But on the other hand it's the simplest way to avoid duplication between builds list and build details pages, so let's keep it.

Reply:
Yeah, it's not entirely perfect, but as you say.  (For the future, it might be better to explicitly pass down `isLinked` as an option to `BuildsList`.)